cnf-certification-test/accesscontrol/suite.go

```diff
@@ -408,3 +415,44 @@ func TestAutomountServiceToken(env *provider.TestEnvironment, testerFuncs rbac.A
 		ginkgo.Fail(fmt.Sprintf("% d pods that failed automount test", n))
 	}
 }
+
+func TestOneProcessPerContainer(env *provider.TestEnvironment) {
+	var badContainers []string
+
+	for _, cut := range env.Containers {
+		debugPod := env.DebugPods[cut.NodeName]
+		if debugPod == nil {
+			ginkgo.Fail(fmt.Sprintf("Debug pod not found on Node: %s", cut.NodeName))
+		}
+
+		ocpContext := clientsholder.Context{
+			Namespace:     debugPod.Namespace,
+			Podname:       debugPod.Name,
+			Containername: debugPod.Spec.Containers[0].Name,
+		}
+
+		pid, err := crclient.GetPidFromContainer(cut, ocpContext)
+		if err != nil {
+			tnf.ClaimFilePrintf("Could not get PID for: %s, error: %s", cut.StringShort(), err)
+			badContainers = append(badContainers, cut.Data.Name)
```

Comment:
use cut.String() instead of cut.Data.Name

Reply:
Done